examples/build_overrides/build.gni

```diff
@@ -15,4 +15,5 @@
 declare_args() {
   # Root directory for build files.
   build_root = "//third_party/connectedhomeip/build"
+  build_java_matter_controller = true
```

Comment:
We can not hard set build_java_matter_controller to true, this will break Android CHIPTool example, right?

Use name build_java_api instead of build_java_matter_controller

Reply:
java_api is also used in android app, it is better to specify as build_java_matter_controller

Reply:
just set it as false, thanks